client/src/store/modules/posts.js

```diff
@@ -84,6 +89,20 @@ const actions = {
         commit('setLoading', false)
       })
   },
+  async getUserLikedPostsAction ({ commit }, payload) {
+    commit('setLoading', true)
+    commit('clearUserPosts')
+    getUserLikedPostsById(payload)
```

Comment:
I'd suggest adding new line before promise (for readability - currently it looks like same chaining process).
And using 'then' inside 'async' when 'await is available - is kinda ugly;)